recipes/gemviz/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "gemviz" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "740b4be1938320efd84ab6845aadaeb71c3047de1c927625613539e30b6e844a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

No need to template something that will be used only once.

```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: 740b4be1938320efd84ab6845aadaeb71c3047de1c927625613539e30b6e844a
```

---

_au contraire_, this definition has value to the maintainer.  It places the version and sha256 adjacent to each other, reducing the chance of inconsistency errors experienced in previous submissions.  Is removal absolutely necessary?

---

We have a bot that updates the recipe. We don't update that manually ever.

---

Most often in the recipes I maintain, they are update be me, not by bot.  The bot misses some development details.